mapper/orm/src/main/java/org/hibernate/search/mapper/orm/cfg/HibernateOrmMapperSettings.java

```diff
@@ -296,6 +323,17 @@ private IndexingPlanRadicals() {
 		public static final String SYNCHRONIZATION_STRATEGY = "synchronization.strategy";
 	}
 
+	/**
+	 * Configuration property keys without the {@link #PREFIX prefix} + {@link Radicals#INDEXING_LISTENERS_PREFIX}.
+	 */
+	public static final class IndexingListenersRadicals {
```

Comment:
Might make more sense to merge `IndexingPlanRadicals` and this one under a single `IndexingRadicals`? You can do so without caring about backwards compatibility since `IndexingPlanRadicals` was introduced in 6.2.